src/app/dashboard/layout.tsx

```diff
@@ -20,7 +20,7 @@ const queryClient = new QueryClient({
       refetchOnMount: false,
       refetchOnReconnect: false,
       refetchOnWindowFocus: false,
-      gcTime: 3600000,
+      gcTime: 300000,
```

Comment:
reducing to 5 minutes to clear up memory more often as we have orphaned keys over time due to how react-query recommends to depend on other data (by creating new keys)